lib/fake_data.rb

```diff
@@ -0,0 +1,114 @@
+require "csv"
+
+class FakeData
+  def self.insert(proportion: 1)
+    new.insert(proportion)
+  end
+
+  def self.delete
+    new.delete
+  end
+
+  def insert(proportion)
+    if has_test_subscribers?
+      puts "There is already test data in the system."
+      puts "Run rake fake_data:delete first."
+      return
```

Comment:
Quite minor but probably want to raise here so it has an exit code other than 0. Otherwise may look successful running it when it's not.

Reply:
Good point! Fixed and rebased!